src/pages/Explorer/Layouts/SheetsTemplate.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+    import Svg from 'webkit/ui/Svg/svelte'
+
+    export let item;
+
+    let isOpen = false;
+</script>
+
+<div class="{isOpen ? 'open' : 'close'}" on:click={() => isOpen = !isOpen}>
+    <div class="row justify v-center">
+        <h5>{item.title}</h5>
+        <Svg id='arrow-down' w="10" class="$style.arrow" />
+    </div>
+    {#if isOpen}
+        <div class="mrg-l mrg--t mrg--b description">
+            <p class="mrg-s mrg--b">
+                {item.desc}
+            </p>
+            <a href={item.url} target="_blank" class="btn-1 btn--s row v-center c-white">
+                <div class="mrg-s mrg--r">Open template</div>
+                <Svg id='external-link' w="12" class="$style.link" />
```

Comment:
`c-white` and `$style.link` are not needed due to `btn-1` class (see the webkit's `Button` story)